src/core/DisplayGroup.h

```diff
@@ -58,6 +70,7 @@ class DisplayGroup : public Coordinates,
         public boost::enable_shared_from_this<DisplayGroup>
 {
     Q_OBJECT
+    Q_PROPERTY( bool showWindowTitles READ getShowWindowTitles CONSTANT )
```

Comment:
This property is not constant, it can change because it has a setter. Even if we don't use it right now because we always replace the entire DisplayGroup on the WallWindow side, this feels incorrect and I would add the WRITE and NOTIFY properties.